script.js

```diff
@@ -0,0 +1,559 @@
+const mainTeacherBlock = document.createElement('div');
```

Comment:
- [ ] I can see that you put a lot of effort into creating this script, but according to the project requirement, you should use a JavaScript variable with the data about the speakers and use it when the page loads. In order to do this, you should do the following:
- [ ] The list of speakers used to generate the HTML is assigned to a variable as an array or object.
- [ ] The list of speakers contains a speaker object for each that holds their specific data, for instance:
```
const speakerList = [ { name: "Speaker Name",... } ]
```
- [ ] The object contains all the information needed to generate the dynamic element, at least the name, role, description, and image URL about the speaker. For example:
```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```
- [ ] There is a function that iterates through the speaker's list on window load and creates HTML elements with the data from each object. A good approach here would be using the [map function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map).